src/pybind/mgr/mgr_module.py

```diff
@@ -29,13 +29,13 @@
 if sys.version_info >= (3, 8):
     from typing import get_args, get_origin
 else:
-    def get_args(tp):
+    def get_args(tp: Any) -> Any:
```

Comment:
with this change, we trade the `sys.version_info >= (3, 8)` for this one. not sure it's always a bargain or not though.

Reply:
centos 8 comes with python3.6.8 .

Reply:
sorry, i was wrong. after this change both branches are tested on a testbed with python>=3.8 .